infra/gcp/ensure-prod-storage.sh

```diff
@@ -342,6 +342,12 @@ color 6 "Handling special cases"
         "k8s-prow-builds.svc.id.goog[test-pods/k8s-infra-gcr-promoter-test]" \
         "${GCR_BACKUP_TEST_PRODBAK_PROJECT}" \
         $(svc_acct_email "${GCR_BACKUP_TEST_PRODBAK_PROJECT}" "${PROMOTER_SVCACCT}")
+
+    color 6 "Empowering promoter-scanning namespace to use prod promoter svcacct"
+    empower_ksa_to_svcacct \
+        "k8s-prow-builds.svc.id.goog[test-pods/k8s-infra-gcr-promoter-scanning]" \
```

Comment:
can you add another binding for `k8s-infra-prow-build-trusted.svc.id.goog[test-pods/k8s-infra-gcr-promoter-scanning`? I would like us to eventually migrate over to that cluster

Reply:
done